CHANGELOG.md

```diff
@@ -54,6 +54,7 @@
 - Improve collapsing of whitespace when pasting to slate text block @tiberiuichim
 - Avoid warning for missing value in NumberWidget @tiberiuichim
 - Fix crash in Slate link editing in a dexterity field @tiberiuichim
+- Fix select widget loosing focus when the value has changed @reebalazs
```

Comment:
@reebalazs For future PRs, please note that the changelog now uses towncrier. That will help avoid putting the release note under the wrong release, like here.

Reply:
Sorry for that - old habits leave on :)